tests/phpunit/elementor/modules/image-loading-optimization/test-module.php

```diff
@@ -12,6 +12,47 @@
  * @group image-loading-optimization
  */
 class Elementor_Image_Loading_Optimization_Test_Module extends Elementor_Test_Base {
+	/**
+	 * An array of attachments.
+	 *
+	 * @type int[]
+	 */
+	static $attachments = [];
+
+	/**
+	 * Setup shared attachments.
+	 */
+	public static function set_up_before_class() {
```

Comment:
In order to add tests for multiple layout use cases, it would be nice to have a set of test images that we can set up during these tests. Do you all have any preferred way of adding test images to the test suite? I've set up the process here that should make creating these attachments easier during a test run, but didn't add any more image files until I could get feedback from you.

Reply:
@joemcgill I am sure you can add more images to the resources folder 👍